src/App/Module.php

```diff
@@ -251,16 +251,16 @@ public function run(Core\L10n\L10n $l10n, App $app, LoggerInterface $logger, arr
 
 		call_user_func([$this->module_class, 'init'], $this->module_parameters);
 
-		// "rawContent" is especially meant for technical endpoints.
-		// This endpoint doesn't need any theme initialization or other comparable stuff.
-		call_user_func([$this->module_class, 'rawContent'], $this->module_parameters);
-
 		if ($server['REQUEST_METHOD'] === 'POST') {
 			Core\Hook::callAll($this->module . '_mod_post', $post);
 			call_user_func([$this->module_class, 'post'], $this->module_parameters);
 		}
 
 		Core\Hook::callAll($this->module . '_mod_afterpost', $placeholder);
 		call_user_func([$this->module_class, 'afterpost'], $this->module_parameters);
+
+		// "rawContent" is especially meant for technical endpoints.
+		// This endpoint doesn't need any theme initialization or other comparable stuff.
+		call_user_func([$this->module_class, 'rawContent'], $this->module_parameters);
```

Comment:
As far as I know we haven't had any other module implement both a `post` method and a `rawContent` method, otherwise we would have witnessed the issue I got: Issue a POST request, get the GET result and no processing happens.